docs/api/saved-scenarios.md

```diff
@@ -95,8 +98,16 @@ Authorization: Bearer YOUR_TOKEN
     "source": "ETM",
     "balanced_values": {},
     "metadata": {},
+    "active_couplings": [],
     "start_year": 2019,
-    "owner": null,
+    "inactive_couplings": [],
+    "users": [
+      {
+        "id": 123,
+        "email": null,
```

Comment:
Indeed! We also give the users back with the scenario. Let's investigate why, and if we can get rid of the email (recommended) or if we can only show emails when somebody owns the scenario. 

For now let's not advocate here that we are showing emails around 😬 

Very nice catch